src/quo2/components/links/link_preview/style.cljs

```diff
@@ -0,0 +1,43 @@
+(ns quo2.components.links.link-preview.style
+  (:require
+    [quo2.foundations.colors :as colors]))
+
+(defn container
+  [preview-enabled?]
+  (merge {:border-width       1
+          :border-color       (colors/theme-colors colors/neutral-20 colors/neutral-80)
+          :background-color   (colors/theme-colors colors/white colors/neutral-80-opa-40)
+          :border-radius      16
+          :padding-horizontal 12
+          :padding-top        10
+          :padding-bottom     12}
+         (when-not preview-enabled?
+           {:height          139
+            :align-items     :center
+            :justify-content :center})))
+
+(def header-container
+  {:flex-direction :row
+   :align-items    :center})
+
+(def title
+  {:flex          1
+   :margin-bottom 2})
+
+(defn link
+  []
+  {:margin-top 8
+   :color      (colors/theme-colors colors/neutral-50 colors/neutral-40)})
+
+(defn thumbnail
+  [size]
+  {:width         "100%"
```

Comment:
FYI: Images are tricky beasts, they don't obey the laws of flexbox that well. Without this `:width "100%"` no flex property that I tried enlarged the image up to the full parent width.

Reply:
Yep :/ did you try working with aspect ratio?

Reply:
Yeah, I tried all sorts of things with the aspect ratio, resize mode, object fit and flexbox props :shrug:

Reply:
FMI :)

Reply:
> FMI :)

Foster-Miller, Inc?